qiskit/providers/ibmq/ibmqbackendservice.py

```diff
@@ -182,24 +183,29 @@ def jobs(
             else:
                 raise IBMQBackendValueError('unrecognized value for "status" keyword '
                                             'in job filter')
-            api_filter.update(this_filter)
+            filters.append(this_filter)
 
         if job_name:
-            api_filter['name'] = {"regexp": job_name}
+            filters.append({'name': {"regexp": job_name}})
 
-        # Create the date query according to the parameters specified.
-        if start_datetime or end_datetime:
-            date_filter = {'creationDate': {}}  # type: ignore[var-annotated]
-            if start_datetime:
-                date_filter['creationDate'].update({'gt': start_datetime.isoformat()})
-            if end_datetime:
-                date_filter['creationDate'].update({'lt': end_datetime.isoformat()})
-
-            api_filter.update(date_filter)
+        if start_datetime:
```

Comment:
Can you remove the example of using `db_filter` to filter date in the docstring?